app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -318,7 +318,12 @@ class PipelineSampleReads extends React.Component {
     const csv_background_id_param = param_background_id
       ? `?background_id=${param_background_id}`
       : defaultBackground;
```

Comment:
this code seems kind of dangerous in its use of '?' and '&' -- is there a way to write this that clearly puts only one ? and the rest of the parameters joined by &? i'm imagining the use of an array and join.('&')